clang/lib/Analysis/FlowSensitive/DataflowEnvironment.cpp

```diff
@@ -556,6 +763,10 @@ void Environment::pushCallInternal(const FunctionDecl *FuncDecl,
     const VarDecl *Param = *ParamIt;
     setStorageLocation(*Param, createObject(*Param, Args[ArgIndex]));
   }
+
+  ResultObjectMap = std::make_shared<PrValueToResultObject>(
```

Comment:
I wonder if this is something that we might want to get cached. That being said, we'd probably want to benchmark first.

Reply:
True -- if we're making several calls to the same function, it might indeed be worth caching this.

I think this isn't a priority for the time being though because a) I'm not aware of any non-test models that use context-sensitive analysis yet, and b) as you say, we should benchmark to see whether this makes up a significant portion of the total effort for analyzing the callee (which in itself will be expensive).